src/arrow-sdk.ts

```diff
@@ -115,13 +117,15 @@ export const bytecodeHashes: any = {
 export async function estimateOptionPrice(option: Option, version: VERSION = VERSION.V2) {
     let strike = undefined
     switch(version) {
-        case VERSION.V2: {
+        case VERSION.V2: 
             strike = option.strike
-        }
+            break;
         case VERSION.V3:
-        case VERSION.COMPETITION: {
+        case VERSION.COMPETITION: 
             strike = (option.strike as number[]).join('|')    
-        }
+            break;
+        default:
```

Comment:
This is unreachable code in typescript (the function will force users to pass a VERSION enum which only has three options). Do you still think we should include it?

Reply:
Also, if we do choose to include it, it would also mean we need to add this check on VERSION for each function in the SDK.

Reply:
Spoke to Osama and he said he would keep it although its may be unreachable in some cases. I will add the checks where appropriate.